back/db/migrate/20230718214736_add_two_initiative_statuses_and_update_colors.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+class AddTwoInitiativeStatusesAndUpdateColors < ActiveRecord::Migration[7.0]
+  class StubInitiativeStatus < ActiveRecord::Base # rubocop:disable Rails/ApplicationRecord
+    self.table_name = 'initiative_statuses'
+  end
+
+  def change
+    reversible do |dir|
+      dir.up do
+        service = MultilocService.new
+
+        proposed = StubInitiativeStatus.find_by(code: 'proposed')
+        return unless proposed
+
+        proposed.ordering = 150
+        proposed.color = '#BEE7EB'
+        proposed.save!
```

Comment:
```suggestion
        StubInitiativeStatus.find_by(code: 'proposed')&.update!(ordering: 150, color: '#BEE7EB')
```
It's totally fine as it is now, just the next time you could use a simpler version.